components/dialog/dialog-fullscreen.js

```diff
@@ -209,12 +209,18 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 	render() {
 
 		const heightOverride = {} ;
+		let topOverride = null;
 		if (this._ifrauContextInfo) {
 			// in iframes, use calculated available height from dialog mixin minus padding
 			heightOverride.height = mediaQueryList.matches
 				? `${this._ifrauContextInfo.availableHeight - 42}px`
 				: `${this._ifrauContextInfo.availableHeight - 60}px`;
 			heightOverride.minHeight = heightOverride.height;
+			const iframeTop = this._ifrauContextInfo.top < 0
+				? -this._ifrauContextInfo.top
```

Comment:
If the contextInfo.top is positive, that means we are scrolled above the iframe. So, set the top of the iframe to 0 (we can't go above it). Otherwise, when the context.top is negative, we flip the sign to get the correct top.

Reply:
Haha iframes are fun eh.